<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicatorDialog</class>
 <widget class="QDialog" name="CommunicatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9,1,0">
      <item>
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipLine">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portSpin">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>4723</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,3">
      <item>
       <widget class="QLabel" name="imLabel">
        <property name="text">
         <string>I'm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="clientRB">
        <property name="text">
         <string>Client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="serverRB">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="OKButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiting for connection ...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
